docs/requirements-doc.txt

```diff
@@ -1,19 +1,18 @@
 autodocsumm
-docutils==0.16
+docutils
 myst-parser
 packaging   # required by micropip at import time
-sphinx>=4.4.0
-sphinx-argparse-cli~=1.6.0
-sphinx_book_theme
-sphinx-issues==2.0.0
-sphinx-js==3.2.1
-sphinx-version-warning~=1.1.2
-sphinx-panels
+sphinx>=5.3.0
+sphinx-argparse-cli>=1.6.0
+sphinx_book_theme>=0.4.0rc1
+sphinx-issues
+sphinx-js>=3.2.1
+sphinx-version-warning>=1.1.2
```

Comment:
BTW, `sphinx-version-warning` is not maintained and it has a bug that requires [root permission to run](https://github.com/humitos/sphinx-version-warning/issues/22), so probably we should replace it someday.